nuimo/src/main/kotlin/com/senic/nuimo/NuimoBluetoothController.kt

```diff
@@ -58,6 +61,13 @@ class NuimoBluetoothController(bluetoothDevice: BluetoothDevice, context: Contex
         reset()
     }
 
+    fun rebootToDfuMode() {
```

Comment:
It's convient to have this method return false when
- nuimo isn't connected
- rebootToDfuModeCharacteristic is nil

this also allows accessing `rebootToDfuModeCharacteristic` without `?`